pkg/gardenadm/cmd/bootstrap/bootstrap.go

```diff
@@ -42,7 +43,7 @@ gardenadm bootstrap --kubeconfig ~/.kube/config`,
 	return cmd
 }
 
-func run(_ context.Context, ioStreams genericiooptions.IOStreams, _ *Options) error {
-	fmt.Fprintln(ioStreams.Out, "not implemented as well")
+func run(_ context.Context, globalOpts *cmd.Options, _ *Options) error {
+	fmt.Fprintln(globalOpts.IOStreams.Out, "not implemented as well")
```

Comment:
Should we use the logger here to start with a good example? (we only want to use `fmt.Fprintln` for non-log output)

Reply:
Yes